### PR TITLE
Homework media queries

### PR DESCRIPTION
**Pull request by przemekjuzwiszin**
Hey guys and ladies:) I fink that code is good. I now is Christmas time but please check this little work that I made. And enjoy:) Keep doing the good job and thanks for spending your time to check on my code:) Regards to team Frontend Company.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
       <nav class="navigation">
         <ul class="navigation__list">
           <li class="navigation__item">
-            <a class="navigation__link" href="index.html">Strona główna</a>
+            <a class="navigation__link navigation__link--return" href="index.html">Strona główna</a>
           </li>
           <li class="navigation__item">
```
Comment (karbowskam) [inline index.html:18]: Hm, where is </li>? :) I can't see closing tag

Comment (karbowskam) [reply index.html:18]: I mean where is "</li>" :)?

Comment (karbowskam) [reply index.html:18]: I cant write down this closing tag, it change my word to white space. I can't see "/li"

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (robson60000) [inline .vscode/settings.json:1]: You should add entire .vscode folder to gitignore.
This is VSCode configuration directory and it should not be in the repo

File: index.html (hunk 1 above)
Comment (robson60000) [reply index.html:18]: Do you mean `<li>` from the 18th line?
It is in 20th line. Use this button to expand the code.
image
For pasting the code use this button:
image

**Review by robson60000 — CHANGES_REQUESTED** [review]

**Comment by przemekjuzwiszin** [comment]
Hi:) please check this ignore file. Is it o.k now? Do I need to correct something?

**Review by robson60000 — APPROVED** [review]

**Pull request merged**